cogs/DLP.py

```diff
@@ -0,0 +1,30 @@
+from discord.ext import commands
+import config
+import discordlists
+
+class DiscordListsPost(commands.Cog, name="DLP"):
+    def __init__(self, bot):
+        self.bot = bot
+        self.api = discordlists.Client(self.bot)  # Create a Client instance
+        self.api.set_auth("discordextremelist.xyz", config.DELTOKEN) # Set authorisation token for a bot list
```

Comment:
Imagine not using del.py smh

Reply:
Sssshhh i will get to this some other time